_data/other.yml

```diff
@@ -328,6 +328,13 @@ websites:
       exceptions:
           text: "SMS only available as a backup to Google Authenticator."
 
+    - name: SignNow
+      url: https://www.signnow.com/
+      img: signnow.png
+      tfa: No
+      twitter: signnow
+      facebook: signnow
```

Comment:
It looks like their Facebook page is actually `BarracudaSignNow`. Not sure why it isn't correctly linked on their site...